src/components/StorySection.js

```diff
@@ -0,0 +1,62 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { containsHtml, trimNewLines } from '../helpers';
+import { Button } from './Button';
+import { HtmlView } from './HtmlView';
+import { ImageSection } from './ImageSection';
+import { MediaSection } from './MediaSection';
+import { BoldText } from './Text';
+import { Wrapper, WrapperHorizontal } from './Wrapper';
+
+// eslint-disable-next-line complexity
+export const StorySection = ({ contentBlock, index, openWebScreen, settings }) => {
+  return (
+    <>
+      {index > 0 && !!contentBlock.title && (
+        <Wrapper>
+          <BoldText>{trimNewLines(contentBlock.title)}</BoldText>
+        </Wrapper>
+      )}
+      {!!contentBlock.intro && (
+        <WrapperHorizontal>
+          <HtmlView
+            html={trimNewLines(
+              `<div>${
+                containsHtml(contentBlock.intro)
+                  ? contentBlock.intro
+                  : `<p>${contentBlock.intro}</p>`
+              }</div>`
+            )}
+            tagsStyles={{ div: { fontFamily: 'titillium-web-bold' } }}
+            openWebScreen={openWebScreen}
+          />
+        </WrapperHorizontal>
+      )}
+      {index > 0 && !!contentBlock.mediaContents && (
```

Comment:
this had a comment before as well

> // skip images for the first content block because they are rendered as main images

do you think it is unnecessary?

Reply:
hmm. depends on how you reach the file and look through it. and since it depends on that, i reintroduced the comments ;)